upgrading/automated_upgrades.adoc

```diff
@@ -1002,7 +748,7 @@ master-etcd-master1.example.com          1/1       Running   6          5d
 ----
 
 . Verify that you are running the expected versions of the *docker-registry*
-and *router* images, if deployed.
+and *router* images, if deployed:
```

Comment:
Seeing from the pre-review draft, the following check command is still using v3.10 image tag, should be updated to v3.11.

Reply:
Also the output of `oc get nodes` is showing v1.10, should be some v1.11 string.

Reply:
@jianlinliu, I'm adding  a v.1.11 string, though I'll be happy to change it if you have a better value, and updated the image tag. Please let me know if you hear what the latest version tag will be.

Reply:
@kalexand-rh The first version of 3.11 image tag would be v3.11.16, and `oc get nodes` is showing "v1.11.0+d4cacc0"